testdata/xml-validate/invalid-data.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<knora xmlns="https://dasch.swiss/schema"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/resources/schema/data.xsd"
+       shortcode="9999"
+       default-ontology="onto">
+
+    <!-- Cardinality Violation -->
+
+    <resource label="Max Cardinality Violation" restype=":SubClass" id="max-card-violation">
+        <text-prop name=":hasSimpleText">
+            <text encoding="utf8">Text Value One</text>
+            <text encoding="utf8"
+                  comment="This property has a cardinality 1"
+            >Text Value Two</text>
+        </text-prop>
+    </resource>
+
+
+    <!-- Ontology Violations -->
+
+    <resource label="Link Value Target" restype=":NormalClass" id="link-value-target-wrong"/>
+
+    <resource label="Target Link Class Mismatch" restype=":NormalClass" id="link-value-wrong">
+        <resptr-prop name=":linkProp">
+            <resptr comment="The target of the link should be of the class :CardOneResource"
+            >link-value-target-wrong</resptr>
+        </resptr-prop>
+    </resource>
+
+
+    <resource label="Link Value Target" restype=":SubClass" id="link-value-target-correct">
+        <text-prop name=":hasSimpleText">
+            <text encoding="utf8"
+                  comment="This is correct and should not appear in the validation errors."
+            >Text</text>
+        </text-prop>
+    </resource>
+
+    <resource label="Target Link Class Correct" restype=":NormalClass" id="link-value-correct">
+        <resptr-prop name=":linkProp">
+            <resptr comment="This is correct and should not appear in the validation errors."
+            >link-value-target-correct</resptr>
+        </resptr-prop>
+    </resource>
```

Comment:
is it necessary to have "correct cases" here?

Reply:
I wanted to have a class that was not itself problematic. The reason why I included it as correct, is because the `ObjectClassConstraint` of the property is the superclass of `:SubClass`. This is here to ensure that this will not fail in the validation.

Reply:
ok